testing/e2e/app/routes/InformationPanel/spec.ts

```diff
@@ -0,0 +1,45 @@
+import { test, expect, type Page } from '@playwright/test';
+
+test.describe('Information Panel Resizing', () => {
+  test('should adjust information panel width', async ({ page }) => {
+    await page.goto('/InformationPanel', { waitUntil: 'domcontentloaded' });
+    const infoPanel = page.locator('#InformationPanel');
+    const initialWidth = (await infoPanel.boundingBox())?.width;
+    expect(await infoPanel.getAttribute('style')).toBeFalsy();
+
+    await resizeInformationPanel(page);
+
+    const newWidth = (await infoPanel.boundingBox())?.width;
+    expect(newWidth).toBe((initialWidth as number) - 100);
+    expect(await infoPanel.getAttribute('style')).toBeTruthy();
+  });
+  test('should reset styles on orientation change', async ({ page }) => {
+    await page.goto('/InformationPanel', { waitUntil: 'domcontentloaded' });
+    const infoPanel = page.locator('#InformationPanel');
+    expect(await infoPanel.getAttribute('style')).toBeFalsy();
+
+    await resizeInformationPanel(page);
+
+    expect(await infoPanel.getAttribute('style')).toBeTruthy();
+    await page.click('#OrientationButton');
+    expect(await infoPanel.getAttribute('style')).toBeFalsy();
+  });
+
+  async function resizeInformationPanel(page: Page) {
+    const resizer = page.locator('._iui3-resizer');
```

Comment:
Targeting these `_iui3` classes is a no-go. It's an implementation detail that should never be used anywhere.

You can instead try to find the edge of the panel and hover/mousedown on it.

Reply:
Okay, I'll see if I can find the left edge of the panel instead.

Reply:
Done in this [commit.](https://github.com/iTwin/iTwinUI/pull/1964/commits/f1ffdde84c52d4237d06660be4b449c56ddd3db9)